src/Lykke.Service.BlockchainWallets/Workflow/CommandHandlers/BeginTransactionHistoryMonitoringCommandHandler.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Net;
+using System.Runtime.InteropServices.ComTypes;
+using System.Threading.Tasks;
+using Common.Log;
+using JetBrains.Annotations;
+using Lykke.Cqrs;
+using Lykke.Service.BlockchainApi.Client;
+using Lykke.Service.BlockchainWallets.Core;
+using Lykke.Service.BlockchainWallets.Core.Repositories;
+using Lykke.Service.BlockchainWallets.Core.Services;
+using Lykke.Service.BlockchainWallets.Workflow.Commands;
+
+
+namespace Lykke.Service.BlockchainWallets.Workflow.CommandHandlers
+{
+    [UsedImplicitly]
+    public class BeginTransactionHistoryMonitoringCommandHandler
+    {
+        private readonly IBlockchainIntegrationService _blockchainIntegrationService;
+        private readonly IMonitoringSubscriptionRepository _monitoringSubscriptionRepository;
+        private readonly ILog _log;
+
+
+        public BeginTransactionHistoryMonitoringCommandHandler(
+            IBlockchainIntegrationService blockchainIntegrationService,
+            IMonitoringSubscriptionRepository monitoringSubscriptionRepository,
+            ILog log)
+        {
+            _blockchainIntegrationService = blockchainIntegrationService;
+            _monitoringSubscriptionRepository = monitoringSubscriptionRepository;
+            _log = log;
+        }
+
+        [UsedImplicitly]
+        public async Task<CommandHandlingResult> Handle(BeginTransactionHistoryMonitoringCommand command, IEventPublisher publisher)
+        {
+            _log.WriteInfo(nameof(BeginTransactionHistoryMonitoringCommand), command, "");
```

Comment:
Will be better to update Lykke.Cqrs nuget and remove logging of messages (and message handling errors) at all. See https://github.com/LykkeCity/Lykke.Job.BlockchainCashinDetector/compare/test...LWDEV-7550-deposits-became-stuck#diff-1dcd8f3883fc4a8d901019b6ce2fb514R70